src/app/modules/feedback/components/file-ticket/file-ticket.component.ts

```diff
@@ -76,4 +88,10 @@ export class FileTicketComponent {
     this.feedbackService.showTicketSuccessMsg(ticketUrl);
     this.dialogRef.close();
   }
+
+  private getSystemFileSizeLimit(): void {
```

Comment:
Maybe we should have `getImagesValidator` accept a plain number instead of an observable and have functions that would attach a validator once `support.attach_ticket_max_size` was loaded.